pages/company/visualizations/company_core_contributors.py

```diff
@@ -0,0 +1,288 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import color_seq
+from queries.company_query import company_query as cq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+
+PAGE = "company"
+VIZ_ID = "company-core-contributors"
+
+paramter_1 = "core-contributions-required"
+paramter_2 = "checks"
+paramter_3 = "core-contributors-required"
+
+
+gc_compay_core_contributors = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Company Core Contributors",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody(
+                            "This graph counts the number of core contributions that COULD be linked to each company.\n\
+                            The methodology behind this is to take each associated email to someones github account\n\
+                            and link the contributions to each as it is unknown which initity the actvity was done for.\n\
+                            Then the graph groups contributions by contributors and filters by contributors that are core.\n\
+                            Contributions required is the amount of contributions necessary to be consider a core contributor\n\
+                            Core Contributors required is the amount of core contributors needed to have the domain listed."
+                        ),
```

Comment:
"This graph looks at all of the email domains a user has contributed with. For instance, a contributor might have used '[johndoe@gmail.com](mailto:johndoe@gmail.com)' for some contributions and '[jdoe12@yahoo.com](mailto:jdoe12@yahoo.com)' for others. We count both of those domain uses once. ALSO FILTERS USERS BY THE VOLUME OF THEIR CONTRIBUTIONS. We acknowledge that we'll likely multi-count affiliations for useres so an assumption that most people don't contribute under many different company emails is necessary." - rough thoughts on graph info. I don't know how to include the filtering-users thing in the explanation, will have to edit this later.

Reply:
We'll save this for the next PR.